controllers/MessageController.ts

```diff
@@ -59,6 +60,11 @@ export default class MessageController implements MessageControllerI {
 
     }
 
+    findMessageById = (req:Request, res:Response) =>{
```

Comment:
Missing comments for JSdoc, can you please add that? LGTM otherwise.

Reply:
Added JSdoc for the new message api.